hphp/runtime/vm/jit/retranslate-all.md

```diff
@@ -0,0 +1,108 @@
+Overview
+--------
+
+The retranslate all feature performs many retranslations simultaneously. These
+threads need space to translate into before statically relocate to the final
```

Comment:
`threads need space to translate into before they statically relocate to the final`